src/advanced-slots.tsx

```diff
@@ -0,0 +1,350 @@
+                </Text2>
+            ))
+        ) : (
+            <Text2 regular color={vars.colors.textSecondary}>
+                {description}
+            </Text2>
+        );
+
+    return (
+        <div className={styles.column}>
+            {title && (
+                <Text3 regular color={vars.colors.textPrimary}>
+                    {title}
+                </Text3>
+            )}
+
+            {!!description && normalizedDescription}
+        </div>
+    );
+};
+
+interface RowBlockProps {
+    title?: string;
+    stackingGroup?: RendersNullableElement<typeof StackingGroup>;
+    description?: string;
+    'aria-label'?: string;
+}
+
+export const RowBlock: React.FC<RowBlockProps> = ({
+    title,
+    stackingGroup,
+    description,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <div
+            style={{
+                display: 'flex',
+                alignItems: 'center',
+            }}
+            aria-label={ariaLabel}
+        >
+            {(title || stackingGroup || description) && (
+                <>
+                    {stackingGroup && (
+                        <div
+                            className={sprinkles({
+                                display: 'flex',
+                                justifyContent: 'space-between',
+                                flex: 1,
+                                alignItems: 'center',
+                            })}
+                        >
+                            {title && (
+                                <div style={{paddingRight: '32px'}}>
+                                    <Text2 regular>{title}</Text2>
+                                </div>
+                            )}
+                            <div style={{zIndex: '0'}}>{stackingGroup}</div>
+                        </div>
+                    )}
+                    {!stackingGroup && (
+                        <div
+                            className={sprinkles({
+                                display: 'flex',
+                                justifyContent: 'space-between',
+                                flex: 1,
+                                alignItems: 'center',
+                            })}
+                        >
+                            {title && (
+                                <div style={{paddingRight: '32px'}}>
+                                    <Text2 regular as="div">
+                                        {title}
+                                    </Text2>
+                                </div>
+                            )}
+                            <div>
+                                <Text2 regular color={vars.colors.textSecondary}>
+                                    {description}
+                                </Text2>
+                            </div>
+                        </div>
+                    )}
+                </>
+            )}
+        </div>
+    );
+};
+
+interface SimpleBlockProps {
```

Comment:
buggy for long texts 😬
https://tinyurl.com/2gz238s7

Reply:
I think the component should grow without limitation by default. We can include the possibility to add truncation like in cards, but force ellipsis by default is a bad practice in terms of accessibility.

If you need this truncation, include it as a prop and add it to specs @vinrosa 

example of truncation prop
https://tinyurl.com/2gwfsruu